doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,543 @@
+       -D WITH_QUIRC=OFF \
+       -D WITH_TBB=OFF \
+       -D WITH_TIFF=OFF \
+       -D WITH_VTK=OFF \
+       -D WITH_WEBP=OFF \
+       -D CMAKE_USE_RELATIVE_PATHS=ON \
+       -D CMAKE_SKIP_INSTALL_RPATH=ON \
+       -D ENABLE_BUILD_HARDENING=ON \
+       -D ENABLE_CONFIG_VERIFICATION=ON \
+       -D ENABLE_PRECOMPILED_HEADERS=OFF \
+       -D ENABLE_CXX11=ON \
+       -D INSTALL_PDB=ON \
+       -D INSTALL_TESTS=ON \
+       -D INSTALL_C_EXAMPLES=ON \
+       -D INSTALL_PYTHON_EXAMPLES=ON \
+       -D CMAKE_INSTALL_PREFIX=install \
+       -D OPENCV_SKIP_PKGCONFIG_GENERATION=ON \
+       -D OPENCV_SKIP_PYTHON_LOADER=OFF \
+       -D OPENCV_SKIP_CMAKE_ROOT_CONFIG=ON \
+       -D OPENCV_GENERATE_SETUPVARS=OFF \
+       -D OPENCV_BIN_INSTALL_PATH=bin \
+       -D OPENCV_INCLUDE_INSTALL_PATH=include \
+       -D OPENCV_LIB_INSTALL_PATH=lib \
+       -D OPENCV_CONFIG_INSTALL_PATH=cmake \
+       -D OPENCV_3P_LIB_INSTALL_PATH=3rdparty \
+       -D OPENCV_SAMPLES_SRC_INSTALL_PATH=samples \
+       -D OPENCV_DOC_INSTALL_PATH=doc \
+       -D OPENCV_OTHER_INSTALL_PATH=etc \
+       -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
+       -D OPENCV_INSTALL_FFMPEG_DOWNLOAD_SCRIPT=ON \
+       -D BUILD_opencv_world=OFF \
+       -D BUILD_opencv_python2=OFF \
+       -D BUILD_opencv_python3=ON \
+       -D PYTHON3_PACKAGES_PATH=install/python/python3 \
+       -D PYTHON3_LIMITED_API=ON \
+       -D HIGHGUI_PLUGIN_LIST=all \
+       -D OPENCV_PYTHON_INSTALL_PATH=python \
+       -D CPU_BASELINE=SSE4_2 \
+       -D OPENCV_IPP_GAUSSIAN_BLUR=ON \
+       -D WITH_INF_ENGINE=ON \
+       -D InferenceEngine_DIR=<OpenVINO_ROOT_DIRECTORY>/runtime/cmake/ \
+       -D ngraph_DIR=<OpenVINO_ROOT_DIRECTORY>/runtime/cmake/ \
+       -D INF_ENGINE_RELEASE=2022010000 \
+       -D VIDEOIO_PLUGIN_LIST=ffmpeg,gstreamer \
+       -D CMAKE_EXE_LINKER_FLAGS=-Wl,--allow-shlib-undefined \
+       -D CMAKE_BUILD_TYPE=Release <OpenCV_ROOT_REPO_DIRECTORY> && \
+    make && cmake -P cmake_install.cmake
+    @endcode
+
+    OpenCV package is available at `~/build-opencv/install` directory.
+
+@note To compile application that uses OpenCV, the following environment variables should be specified:
+    @code{.bash}
+    export OpenCV_DIR="<OpenCV_INSTALL_DIR>/cmake"
+    export LD_LIBRARY_PATH="<OpenCV_INSTALL_DIR>/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+    export PYTHONPATH="<OpenCV_INSTALL_DIR>/python${PYTHONPATH:+:$PYTHONPATH}"
+    @endcode
+
+## Building on Windows {#tutorial_openvino_install_building_on_windows}
+
+### Prerequisites {#tutorial_openvino_install_building_on_windows_prerequisites}
+
+1.  Install Microsoft Visual Studio
+2.  Install [cmake](https://cmake.org/download/)
+3.  Install OpenVINO according to [the instruction](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html)
+4.  Install [Intel® Media SDK for Windows](https://www.intel.com/content/www/us/en/developer/tools/media-sdk/choose-download-client.html)
```

Comment:
I believe, MediaSDK is optional dependency.

Reply:
It's needed to build OpenCV with `WITH_MFX` flag. This flag was enabled in OpenCV build for OpenVINO.
I believe we need to keep it to get "OpenVINO-like" version of OpenCV.

Reply:
WITH_MFX means MediaSDK. OpenCV could be build and used without MediaSDK, it's an option. As soon as it's not patch of Intel distribution, I propose to stay with minimal list of option and dependencies.

Reply:
From my perspective the purpose of this document is to provide OpenCV users a way to get OpenCV build that behaves exactly like OpenVINO OpenCV build. OpenVINO OpenCV build had a dependency to MediaSDK, so, I believe, it's better to keep it, so OpenCV user who uses it with OpenVINO has the same experience.